src/content/en/updates/2020/12/webassembly.md

```diff
@@ -0,0 +1,434 @@
+expected, except for function inlining. We plan to address the remaining
+issues in the future, but, for now, please use `-fno-inline` to
+disable it when compiling with any `-O` level optimizations, e.g.:
+
+<pre class="devsite-terminal devsite-click-to-copy">
+emcc -g temp.c -o temp.html \
+     -O3 -fno-inline
+</pre>
+
+### Separating the debug information {: #separate-dwarf }
+
+Debug information preserves lots of details about your code, defined
+types, variables, functions, scopes, and locations – anything that might
+be useful to the debugger. As a result, it often can be larger than the
+code itself.
+
+To speed up loading and compilation of the WebAssembly module, you might
+want to split out this debug information into a separate WebAssembly
+file. To do that in Emscripten, pass a `-gseparate-dwarf=…` flag with
+a desired filename:
+
+<pre class="devsite-terminal devsite-click-to-copy">
+emcc -g temp.c -o temp.html \
+     -gseparate-dwarf=temp.debug.wasm
+</pre>
+
+In this case, the main application will only store a filename
+`temp.debug.wasm`, and the helper extension will be able to locate and
+load it when you open DevTools.
+
+When combined with optimizations like described above, this feature can
+be even used to ship almost-optimized production builds of your
+application, and later debug them with a local side file. In this case,
+we’ll additionally need to override the stored URL to help the extension
+find the side file, for example:
+
+<pre class="devsite-terminal devsite-click-to-copy">
+emcc -g temp.c -o temp.html \
+     -O3 -fno-inline \
+     -gseparate-dwarf=temp.debug.wasm \
+     -s SEPARATE_DWARF_URL=file://[local path to temp.debug.wasm]
+</pre>
+
+## To be continued… {: #future-plans }
+
+Whew, that was a lot of new features!
+
+With all those new integrations, Chrome DevTools becomes a viable,
+powerful, debugger not only for JavaScript, but also for C and C++ apps,
+making it easier than ever to take apps, built in a variety of
+technologies and bring them to a shared, cross-platform Web.
+
+However, our journey is not over yet. Some of the things we’ll be
+working on from here on:
+
+- Cleaning up the rough edges in the debugging experience.
+- Adding support for custom type formatters.
+- Working on improvements to the
+  [profiling](/web/tools/chrome-devtools/evaluate-performance/reference) for WebAssembly apps.
+- Adding support for [code coverage](/web/tools/chrome-devtools/coverage) to make it easier to find
+  unused code.
+- Improving support for expressions in console evaluation.
+- Adding support for more languages.
+- …and more!
+
+Meanwhile, please help us out by trying the current beta on your own code and reporting any found
+issues to
+[https://bugs.chromium.org/p/chromium/issues/entry?template=DevTools+issue](https://bugs.chromium.org/p/chromium/issues/entry?template=DevTools+issue).
+
+Stay tuned for future updates!
+
+## Feedback {: #feedback .hide-from-toc }
```

Comment:
DevTools blog use different footer. Refer to here:
https://github.com/google/WebFundamentals/pull/9163/files#diff-cbc2614d28097dd7a29a74aee0e3e6c91f4efe13efbb4fd9db6902f08dd3e3d5R26

Reply:
So... this is a DevTools blog item then? I guess I'm confused, I thought we want to use same generic format as update last year (https://developers.google.com/web/updates/2019/12/webassembly).

Reply:
Added.